core/frontend/services/routing/controllers/unsubscribe.js

```diff
@@ -0,0 +1,31 @@
+const debug = require('ghost-ignition').debug('services:routing:controllers:unsubscribe');
+const path = require('path');
+const megaService = require('../../../../server/services/mega');
+const labsService = require('../../../../server/services/labs');
+const helpers = require('../../../services/routing/helpers');
+
+module.exports = async function unsubscribeController(req, res, next) {
+    debug('unsubscribeController');
+
+    if (!labsService.isSet('members')) {
+        return next();
+    }
+
+    let data = {};
+
+    try {
+        data.member = await megaService.mega.handleUnsubscribeRequest(req);
```

Comment:
Does this method still make sense to be in `mega` service? :thinking: 

I think it does because it means MEGA is in complete control of what it means to be subscribed and how that is represented